### PR TITLE
Add/cbl platform flags

### PR DESCRIPTION
**Pull request by gilkal**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- More framework adjustments to support running SGW/CBL on Debian12.
- Tested on iOS and Android sanity. More changes may be required.

### CODE REVIEW

File: keywords/SyncGateway.py [hunk 1]
```diff
@@ -683,18 +681,20 @@ def stop_sync_gateways(self, cluster_config, url=None):
         shut down the sync gateway at that url
         """
         ansible_runner = AnsibleRunner(cluster_config)
-
+        extra_vars = {}
```
Comment (muntazim-aliCB) [inline keywords/SyncGateway.py:684]: Is this being used? I see that the default argument for `ansible_runner.run_ansible_playbook()` is already an empty  dict, and it doesn't seem like this variable is ever changed?

File: libraries/provision/ansible/playbooks/tasks/create-sync-gateway-user.yml [hunk 2]
```diff
@@ -1,8 +1,3 @@
-# Add sync_gateway user on Centos/linux/ubuntu/debian
```
Comment (muntazim-aliCB) [inline libraries/provision/ansible/playbooks/tasks/create-sync-gateway-user.yml:1]: This passes for the sanity tests I assume, but are there other tests that need a SG user that no longer pass without this?

Comment (gilkal) [reply libraries/provision/ansible/playbooks/tasks/create-sync-gateway-user.yml:1]: The sync_gateway user is created by default in the installation, so I think we don't need it. The problem is that the old Ansible version is not compatible with Debian12, causing this user creation to fail.

File: keywords/SyncGateway.py (hunk 1 above)
Comment (gilkal) [reply keywords/SyncGateway.py]: Thanks, fixed!

**Review by muntazim-aliCB — APPROVED** [review]

**Pull request merged**